internal/cmd/testdata/TestKeysListCmd/list_all

```diff
@@ -0,0 +1,4 @@
+  NAME        ISSUED FOR  CREATED       EXPIRES       EXTENSION DEADLINE  
+  front-door  admin       4 months ago  never         never               
+  side-door   admin       4 months ago  4 months ago  4 months ago        
+  storage     clerk       4 months ago  4 months ago  never               
```

Comment:
For now these values are stored in separate files.  I think it would be possible to instead store them in package-level variables in the test file, next to the test. That might work better for our use case.

I think that should be possible, I'll see about adding support for that to `gotest.tools/v3/golden`.

I'd also like to change the update flag to `-update` instead of `-test.update-golden`, to make it easier to remember.